templates/view_profile.html

```diff
@@ -156,29 +156,29 @@ <h1>Mod Packs by {{ profile.username }}</h1>
 </div>
 {% endif %}
 
-{% if len(profile.mods) != 0 %}
+{% if len(mods_created) != 0 %}
```

Comment:
`mods_created` is the same as `profile.mods`, but sorted. How would this change filter out unpublished mods?

https://github.com/KSP-SpaceDock/SpaceDock/blob/c6ed96ddf04302cae66153cf8b66e16624db6eb9/KerbalStuff/blueprints/profile.py#L20